chapters/data-collection.tex

```diff
@@ -1,362 +1,299 @@
+
+\subsection{Questionnaire design for quantitative analysis}
+This book covers surveys designed to yield datasets useful for quantitative analysis. This is a subset of surveys, and there are specific design considerations that will help to ensure the raw data outputs are ready for analysis.
+
+From a data perspective, questions with pre-coded response options are always preferable to open-ended questions (the content-based pilot is an excellent time to ask open-ended questions, and refine responses for the final version of the questionnaire). Coding responses helps to ensure that the data will be useful for quantitative analysis. Two examples help illustrate the point. First, instead of asking ``How do you feel about the proposed policy change?'', use techniques like
+\textbf{Likert scales}\sidenote{\textbf{Likert scale:} an ordered selection of choices indicating the respondent's level of agreement or disagreement with a proposed statement.}. Second, if collecting data on medication use or supplies, you could collect: the brand name of the product; the generic name of the product; the coded compound of the product; or the broad category to which each product belongs (antibiotic, etc.). All four may be useful for different reasons, but the latter two are likely to be the most useful for data analysis. The coded compound requires providing a translation dictionary to field staff, but enables automated rapid recoding for analysis with no loss of information. The generic class requires agreement on the broad categories of interest, but allows for much more comprehensible top-line statistics and data quality checks. Rigorous field testing is required to ensure that answer categories are comprehensive; however, it is best practice to include an \textit{other, specify} option. Keep track of those reponses in the first few weeks of fieldwork; adding an answer category for a response frequently showing up as \textit{other} can save time, as it avoids post-coding. 
+
+There is debate over how individual questions should be identified: formats like \texttt{hq\_1} are hard to remember and unpleasant to reorder, but formats like \texttt{hq\_asked\_about\_loans} quickly become cumbersome.
+We recommend using descriptive names with clear prefixes so that variables 
+within a module stay together when sorted 
+alphabetically.\sidenote{\url{https://medium.com/@janschenk/variable-names-in-survey-research-a18429d2d4d8}}
+ Variable names should never include spaces or mixed cases (all lower case is 
+best). Take care with the length: very long names will be cut off in certain 
+software, which could result in a loss of uniqueness. We discourage explicit 
+question numbering, as it discourages re-ordering, which is a common 
+recommended change after the pilot. In the case of follow-up surveys, numbering 
+can quickly become convoluted, too often resulting in variables names like 
+'ag\_15a', 'ag\_15\_new', 'ag\_15\_fup2', etc.
```

Comment:
This chapter is not about questionnaire design. I think it should be moved to the section called "compatibility with analysis software".

Reply:
@luizaandrade i think of variable naming as part of questionnaire design. let's discuss.

Reply:
Sorry, I meant this paragraph is not about questionnaire design. At the beginning of this subsection, we make a point that questionnaire design is about designing the questions, not about programming the instrument. I agree this paragraph should be in the chapter, but I think it should go on the subsection called *compatibility with analysis software*.